project.clj

```diff
@@ -29,11 +29,9 @@
              :test {:dependencies [[print-foo "1.0.2"]]
                     :src-paths ["test/resources"]}
              :1.8 {:dependencies [[org.clojure/clojure "1.8.0"]
-                                  [org.clojure/clojurescript "1.8.51"]
-                                  [javax.xml.bind/jaxb-api "2.3.1"]]}
+                                  [org.clojure/clojurescript "1.8.51"]]}
```

Comment:
This will causes breakages for ClojureScript. We don't use this directly, but I'm afraid it's needed.